src/write-ui-env-file.ts

```diff
@@ -12,7 +12,7 @@ const configFilePath = path.resolve(outputPath, "env-config.js");
 const project = "seds";
 const region = "us-east-1";
 
-export async function writeUiEnvFile(stage: string, local = false) {
+export async function writeLocalUiEnvFile(stage: string) {
```

Comment:
As I'm thinking this through. I think we could add some conditional logic related to `isLocalstack` and then not need this file at all. The cdk creation of the ssm parameter will be correct regardless of it's context.

Reply:
In fact I think you already sort of did it or most of it. I'll comment there.

Reply:
this code is to create the env-config.js file for the react frontend.